### ISSUE TITLE
Accessibility issues with screen reader and pcmanfm qt navigation

### ISSUE THREAD

**Issue by gigino1987** [lxqt/lxqt#2632]
Hi,
I'm blind user and use screen reader for interact with pc/smartphone.
I tried lxqt many time and I think that is one of the best light desktop for Linux.
Therefore I have some problems when navigating in desktop and file manager using Orca.
The problem is same, when I navigating with arrows it reads only name of first item, other items don't read but reads size and data of first file.
For example, if in desktop there are Computer, Trash and Network it reads Computer in first position, but if I try to move it reads: size xxxxx, Date xxxxxxx and not reads Network and Trash.
Same problem I have in pcmanfm qt application, but probably if I replace with gtk pcmanfm probably solve it but I don't tried. Finally also qterminal is not accessible but I can replace with lxterminal.
I hope that pcmanfm qt can be fixed for blind users.

Thanks a lot

**Comment by tsujan** [comment]
Perhaps because I have no experience in using screen readers, I didn't understand the problem in pcmanfm-qt, although I tried to put myself into your place.

If you are on Desktop and there are no items with the names "size xxxxx", or "Date xxxxxxx" there. where do those two strings come from? Maybe this is caused by a setting in Ocra.

**Comment by tsujan** [comment]
> also qterminal is not accessible

Could you please check if Konsole is accessible?

**Comment by gigino1987** [comment]
Hi,
sorry for my big delay, but I don't tried anymore.
Now, I tried lxqt 2.1.0 on Debian Testing and icon moving problem was solved.
I confirm that Konsole and qterminal are not usable with Orca, but I can replace them with lxterminal.
Another problem is applications menu, how I can open it via keyboard?
Alt+F1 don't work and I must open /usr/share/applications folder via pcmanfm-qt.

Thanks for your job

Gigi

**Comment by stefonarch** [comment]
> Another problem is applications menu, how I can open it via keyboard?

Alt+f1 should work, but shortcut can be changed to Meta (windows key)  or whatever by right click on the menu.

**Comment by gigino1987** [comment]
Ok solved, I'm installed Arch with this DE and it's simply awesome! I only replaced some application with another accessible but now I use LXQT correctly. 

Thanks for job!

**Comment by tsujan** [comment]
BTW, navigating with arrow keys seems OK here, whether on Desktop or inside pcmanfm-qt's window.

Please also note that we can't control how well `orca` works with Qt. It's done on a much lower level than that of LXQt. But I admit that I don't know how it might be improved on the GUI level. So, any advice would be welcome.

**Comment by gigino1987** [comment]
Hi, now it's usable, if you can improve accessibility of qterminal, add a shortcut to reach panel  and adjust notifications to be readable from Orca it can becomes final perfect accessible solution. 
I know that some problem depends from QT but I think that your work is simply perfect.

Thanks a lot

Gigi

**Comment by tsujan** [comment]
You're right: LXQt notifications aren't read by Orca. I have no clue why, because our code doesn't do anything special in this regard, but I opened https://github.com/lxqt/lxqt-notificationd/issues/430.
